backend/application/advertisement.ts

```diff
@@ -48,6 +48,26 @@ export class AdvertisementApplicationService {
       )
     }
   }
+  async getBannerAd() {
```

Comment:
```suggestion
  async getUserAdvertisments() {
```
にしましょうか！（ユーザ側の広告ですよね？？）
→取得してくるのはadvertismentのデータのはずなので